pkg/products/threescale/reconciler.go

```diff
@@ -1625,9 +1625,15 @@ func getMTAccountsToBeCreated(usersIdentity []usersv1.Identity, accounts []Accou
 				Name:    identity.User.Name,
 				OrgName: identity.User.Name,
 			})
+			if identity.Extra["email"] != "" {
+				email = identity.Extra["email"]
+			} else {
+				email = fmt.Sprintf("%s@rhmi.io", identity.User.Name)
+			}
+			emailAddrs = append(emailAddrs, email)
```

Comment:
Hi Brian, I don't see an issue with it, but more of a question, do you think it would be better to have email set as part of AccountDetail object? In here: https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/threescale/types.go#L67
Maybe we could add it as part of future tasks?

Reply:
I'm not sure if its a good place as AccountDetail has multiple users associated. We could experiment with it in future though.

Reply:
Thanks Brian, as for our call yesterday, we have agreed that this is possibly something we can't do atm but might investigate further at some later stage.